inventory/rh-sso/group_vars/rh-sso-hosts.yml

```diff
@@ -6,22 +6,26 @@ ansible_become: True
 list_of_packages_to_install:
   - "@rh-sso7"
 
+sso_protocol: https
 sso_admin_user: admin
 sso_admin_pass: mypa55ss
+sso_keystore_name: default
+sso_keystore_pass: changeit
```

Comment:
While you are at it, make sure these 3 keystore variables are also in `defaults/main.yml` (and aligned on naming)

Reply:
Fixed.